zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -142,6 +143,21 @@ function PageWrapper(props) {
     [],
   );
 
+  const handleScroll = ()=>{
+    if(window.pageYOffset > 70){
```

Comment:
@kamthamc Is there a better way to calculate this? I am not sure about hardcoding value here that @ipriyanshi1708 has done.

Reply:
Why not use ref to the top element and show the button if its not visible 

https://blog.devgenius.io/how-to-tell-if-an-html-element-is-visisble-with-javascript-9f5821056e08

Reply:
@ipriyanshi1708 Please make the changes proposed by @chaitanya-cloudknox above 👆

Reply:
> @ipriyanshi1708 Please make the changes proposed by @chaitanya-cloudknox above 👆
Okay, I am working on that.